test/README.md

```diff
@@ -23,20 +75,20 @@ From the root of the module directory execute the following commands:
 
 ```bash
 # Initialize the terraform providers and context
-bundle exec kitchen setup
+bundle exec kitchen setup tf-module
```

Comment:
This test suite is called `simple-example`, let's update this documentation.

Reply:
fixed. thank you.